sdk/eventhub/azure-eventhub/tests/.gitignore

```diff
@@ -0,0 +1,7 @@
+# amqpproxy
+**/amqpproxy_startup.log
+*.bin
```

Comment:
gitignore here versus at the repo level ?

Reply:
Added it here since we're codeowners at this level and it'll be easier to update if needed. Thoughts?